docs/analyzer-rules/AZFW0014.md

```diff
@@ -0,0 +1,29 @@
+# AZFW0011: Missing Registration for ASP.NET Integration
+
+| | Value |
+|-|-|
+| **Rule ID** |AZFW00014|
+| **Category** |[Usage]|
+| **Severity** |Error|
+
+## Cause
+
+This rule is triggered when the ASP.NET function app is missing registration for the appropriate method.
+
+## Rule description
+
+The ASP.NET function app should have registration for `ConfigureFunctionsWebApplication` method. This rule will be violated when the app is missing this registration.
+
+## How to fix violations
+
+Add registration for `ConfigureFunctionsWebApplication` method in Function App. Refer to <ToDo: Add public Msft docs link>
```

Comment:
Do we have any public docs for this available that I can add here?

Reply:
https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide#aspnet-core-integration

Reply:
Thanks